server/controllers/visits.js

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+var crypto = require('crypto');
+var Boom = require('boom');
+var uuid = require('uuid');
+
+var config = require('../config');
+var log = require('../logger')('server.controllers.visits');
+var queue = require('../work-queue/queue');
```

Comment:
I'd really prefer to publish events for the queue, rather than directly call the queue. I'd like to fix this when we add a global event dispatcher to the app, hopefully next sprint.